File: Aula27.02.2025Algoritmo/medAulaVersao_b.c
#include <stdio.h> //importando a biblioteca de entrada e saída
// função principal do programa
int main() {
  printf("Digite quantos número deseja fazer a média: "); // imprime na tela uma
                                                          // mensagem de escolha
                                                          // de número
  int nMedia;                                             // declara nMedia
  scanf("%d", &nMedia); // recebe o valor de nMedia
  int myNumber[nMedia];
  float media;
  int somaMedia = 0;
  int valor = 0;
  int i = 0;

  for (i = 0; i < nMedia; i++) {
    printf("Digite o %dº número : ", i + 1);
    scanf("%d", &valor);
    myNumber[i] = valor;
    somaMedia = somaMedia + myNumber[i];
    media = somaMedia / (i+1);
    printf("Média dos valores parciais %f\n ", media);
  }
media = somaMedia / nMedia;
printf("Média dos valores %f\n", media); // imprime na tela o valor de n e o valor de p
  return 0;      // fecha o programa
}

File: Aula27.02.2025/medAulaVersao_b.py
n_media = int(input("Digite quantos números deseja fazer a média: "))  # Lê o valor de n para poder saber quantos número iremos realizar a média
soma_media = 0  # Inicializa a soma dos números para fazer a média
    
for i in range(n_media): # Laço de repetição
    valor = int(input(f"Digite o {i + 1}º número: "))  # Lê o valor de cada número
    soma_media += valor  # Adiciona o número à soma
    media = soma_media / (i+1)  # Calcula a média
    print(f"Média dos valores Parciais: {media}")  # Imprime o resultado

    print(f"Média dos valores Final: {media}")  # Imprime o resultado
